examples/example_parametric_components/make_all_parametric_components.py

```diff
@@ -417,6 +417,13 @@ def main():
     )
     all_components.append(component)
 
+    component = paramak.HexagonPin(
+        length_of_side=5,
+        distance=10,
```

Comment:
I know this is an ongoing discussion but I still think "distance" isn't descriptive enough.
I find that "extrude_distance" would be clearer. But I understand that would require to be changed in all components...

Reply:
ok we can change this across the collection sometime, but perhaps not in this PR